### ISSUE TITLE
GIDSignIn signInSilently not work

### ISSUE THREAD

**Issue by onmyway133** [onmyway133/ios-issues#9]
## SOLUTION
Need to configure the GIDSignIn (clientID, ...) like the first time we use `[[GIDSignIn sharedInstance] signIn];`

**Comment by omarojo** [comment]
This solutions didn't work for me. Im using the latest Google SignIn library, and running in iOS 9. [GIDSignIn sharedInstance] hasAuthInKeychain] always returns NO. This happens only when I kill the app (after successfully login In) and then open the app again. [[GIDSignIn sharedInstance] signInSilently]; will return error -4. It seems the normal SignIn doesn't save anything in the Keychain.. or the Keychain gets cleared when I the app relaunches or something like that.

**Comment by nlabhart** [comment]
@omarojo were you able to figure this out? I'm having this exact issue.

**Comment by omarojo** [comment]
Nope, I just decided not to use the library. Is a piece of crap. I dont know how other famouse apps are doing it with this library. 

=====================
Omar Juarez Ortiz
Software Developer
www.omarojo.com
=====================




On Tue, Mar 15, 2016 at 10:11 AM -0700, "Noah Labhart" <notifications@github.com> wrote:












@omarojo were you able to figure this out? I'm having this exact issue.
